portal/models/i18n.py

```diff
@@ -1,4 +1,5 @@
 """Module for i18n methods and functionality"""
+from __future__ import unicode_literals  # isort:skip
```

Comment:
There also appear to be a number of unused imports left behind from the utils refactor:
```
from io import BytesIO
import os
import sys
import tempfile

from polib import pofile
import requests```

Reply:
Done, thanks! Looks like Codacy should've caught those- re-enabled it